proseco/guide.py

```diff
@@ -183,19 +183,34 @@ def search_stage(self, stage):
 
         n_sigma = stage['SigErrMultiplier']
 
+        # For the reasonable mag check, override the SigErrMultiplier to be at least 1.
+        # It may go to 0 in later stages for the spoiler checks.
+        mag_err_mult = max(n_sigma, 1)
```

Comment:
After reviewing the data and this code, I think it makes sense to always use at *least* one mag error when checking against the mag bounds.  But changing SigErrMultiplier to 1 in characteristics for the last two stages would creep out of the scope of this PR because it would effect the spoiler checking on every stage 4 and 5 candidate (probably not with much effect but with possible effect).  So here I've just set a lower bound of 1 on the multiplier used for the "select stars by mag" part of stage selection.